packages/app-store/salesforce/lib/assignmentReasonHandler.ts

```
@@ -8,27 +8,52 @@ export async function assignmentReasonHandler({
+        ...returnObject,
+        assignmentReason: `Salesforce account owner: ${teamMemberEmail}${
+          recordId ? ` (Account ID: ${recordId})` : ""
+        }`,
+      };
     case RoutingReasons.ACCOUNT_LOOKUP_FIELD:
-      const assignmentReason = await handleAccountLookupFieldReason(routingFormResponseId, teamMemberEmail);
+      const assignmentReason = await handleAccountLookupFieldReason(
+        routingFormResponseId,
+        teamMemberEmail,
+        recordId
+      );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)